skyve-core/src/main/java/org/skyve/util/test/TestUtil.java

```diff
@@ -201,6 +203,23 @@ public static <T extends PersistentBean> T updateAttribute(final Module module,
 		return bean;
 	}
 
+	public static List<String> retreiveExcludedUpdateAttributes(Module module, Document document) {
+		String className = String.format("modules.%1$s.%2$s.%2$sFactory", module.getName(), document.getName());
+		Util.LOGGER.fine("Looking for factory class " + className);
+		try {
+			Class<?> c = Thread.currentThread().getContextClassLoader().loadClass(className);
+			if (c.isAnnotationPresent(SkyveFactory.class)) {
+				Util.LOGGER.fine("Found class " + c.getName());
+				SkyveFactory annotation = c.getAnnotation(SkyveFactory.class);
+				return Arrays.asList(annotation.excludedUpdateAttributes());
+			}
+		}
+		catch (Exception e) {
+			System.err.println("Could not find factory class for: " + e.getMessage());
```

Comment:
Maybe change this to a warning or info log? Won't this get thrown every single `testUpdate()` that does not have a factory defined?

Reply:
Thanks, changed to fine logging level to match the other statements. Also added documentation and fixed a typo.